tests/test_interface.py

```diff
@@ -1,12 +1,26 @@
 import pytest
-import ttt.interface as interface
+from ttt.console import Console
 
 
-class Runner():
+class MockConsoleIOErrorInput:
+    def get_input(self, message=""):
+        return "Not a number!"
+
+
+class MockConsoleIOValidInput:
+    def get_input(self, message=""):
+        return "1"
+
+    def print_output(self, output):
+        print(output)
+
+
+class Runner:
     def __init__(self, capfd):
         self._capfd = capfd
 
     def render_board(self, board_state):
+        interface = Console(console=MockConsoleIOValidInput())
```

Comment:
Do you pass a `console` or `io` into the `Console`?

Why do you need to use keyword arguments instead of positional arguments here?